src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1111,11 +1111,21 @@
       "snippetPath": "updateCallNumberSource.sql",
       "fromModuleVersion": "26.1.0"
     },
+    {
+      "run": "after",
+      "snippetPath": "addDcbInstance.sql",
+      "fromModuleVersion": "27.0.0"
+    },
     {
       "run": "after",
       "snippetPath": "addReindexJobResourceName.sql",
       "fromModuleVersion": "26.1.0"
     },
+    {
+      "run": "after",
+      "snippetPath": "addDcbHolding.sql",
+      "fromModuleVersion": "27.0.0"
```

Comment:
"27.0.0" version already released please use version from pom.xml, it's 27.1.0

Reply:
fixed